packages/tsconfig/README.md

```diff
@@ -0,0 +1,23 @@
+# @commercetools-frontend/tsconfig
+
+<p align="center">
+  <a href="https://www.npmjs.com/package/@commercetools-frontend/tsconfig"><img src="https://badgen.net/npm/v/@commercetools-frontend/tsconfig" alt="Latest release (latest dist-tag)" /></a> <a href="https://www.npmjs.com/package/@commercetools-frontend/tsconfig"><img src="https://badgen.net/npm/v/@commercetools-frontend/tsconfig/next" alt="Latest release (next dist-tag)" /></a> <a href="https://bundlephobia.com/result?p=@commercetools-frontend/tsconfig"><img src="https://badgen.net/bundlephobia/minzip/@commercetools-frontend/tsconfig" alt="Minified + GZipped size" /></a> <a href="https://github.com/commercetools/merchant-center-application-kit/blob/main/LICENSE"><img src="https://badgen.net/github/license/commercetools/merchant-center-application-kit" alt="GitHub license" /></a>
+</p>
+
+TSConfig file used by a Custom Application in TypeScript.
+
+## Install
+
+```bash
+$ npm install --save @commercetools-frontend/tsconfig
+```
+
+## Usage
+
+Add to your `tsconfig.json`:
+
+```json
+{
+  "extends": "@commercetools-frontend/tsconfig"
```

Comment:
This should be

```suggestion
  "extends": "@commercetools-frontend/tsconfig/tsconfig.json"
```

However, seeing this makes me wonder if we really need an extra package for it...maybe we can include the `tsconfig.json` file in one of the required packages like `application-config` or `application-shell`... 🤔

Reply:
If we update this example here, I believe we should also update it in the changeset.

Regarding the question raised about whether creating or not a new package, personally I think the `application-config` package would be a good fit to export this configuration file.

Reply:
Done in 6dbbc90aaca54fc841c6aa007e16372be4e3323a